src/components/users/UserDevices.tsx

```diff
@@ -0,0 +1,23 @@
+import { Text } from '@taskany/bricks';
+import { gray9 } from '@taskany/colors';
+import { IconPlusCircleOutline } from '@taskany/icons';
+
+import { PageSep } from '../PageSep';
+import { InlineTrigger } from '../InlineTrigger';
+
+export const UserDevices = () => {
+    return (
+        <div>
+            <Text size="m" color={gray9} weight="bold">
+                Corporate devices
+                <PageSep width={300} margins={5} />
+            </Text>
+
+            <InlineTrigger
+                icon={<IconPlusCircleOutline noWrap size="s" />}
+                text={'Request a device'}
+                onClick={() => {}}
```

Comment:
Probably it's better to disable buttons or inputs not connected yet and leave a TODO comment